recipes/spacexr/meta.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: spacexr
+  version: 2.2.1
+
+source:
+  url: https://github.com/dmcable/spacexr/archive/refs/heads/master.zip
```

Comment:
```suggestion
  url: https://github.com/dmcable/spacexr/archive/0a0861e3d1e16014a20e9b743d0e19d3b42231f3.zip
``

please use this link, it gives you a specific commit of the tool, otherwise you would get the latest master code when we rebuild the package.